api/specs/common/schemas/node-meta-v0.0.1-converted.yaml

```diff
@@ -16,7 +16,7 @@ properties:
   key:
     type: string
     description: distinctive name for the node based on the docker registry path
-    pattern: ^(simcore)/(services)/(comp|dynamic)(/[\w/-]+)+$
+    pattern: ^(simcore)/(services)/(comp|dynamic)(/[^\s/]+)+$
```

Comment:
Why this change in the regex?

Reply:
well. to be honest I just ran the ```make openapi-specs``` recipe.. and did not double-check if what it produces is correct. I can have a look though.

Reply:
added a test to prevent this from happening in the future